home.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Home</title>
+    <link
+      href="http://fonts.googleapis.com/css?family=Lato:300,400,500,600,700"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <section class="main-bg my-5">
+      <div class="container-md mt-5 mb-5">
+        <h1 class="h1 heading">"Hello! Sharing world"</h1>
+        <h1 class="display-1 heading-sub fw-600 text-uppercase">
+          "Jah Malet Fetari" International Chess Tournament 2023
+        </h1>
+        <p class="h4 sub-heading">
+          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Iste fuga
+          nisi laborum nemo delectus iure.
+        </p>
+        <p class="display-4">2023.03.15(WED) ~ 17(FRI)</p>
+        <p class="h5">Ethiopia Science Museum; 4 Kilo, Addis Ababa, Ethiopia</p>
+      </div>
+    </section>
+    <section class="content" id="program">
+      <div class="container-md">
+        <h2 class="text-white text-center py-5">Main Program</h2>
+        <div class="row d-flex justify-content-center program-container"></div>
+        <h2
+          class="text-white text-center py-5 text-decoration-underline fs-5 text-uppercase"
+        >
+          See the whole program
+        </h2>
+      </div>
+    </section>
+    <section class="featured d-flex flex-column" id="speakers">
+      <h2 class="text-black text-center mt-5">Featured Speakers</h2>
+      <hr class="red-hr align-self-center mb-5" />
+      <div
+        class="row container-md justify-content-center mx-auto featured-cards mb-5"
+      ></div>
+    </section>
+    <section class="partner" id="partner">
+      <div class="container-md d-flex flex-column">
+        <h1 class="align-self-center mt-5 mb-2">Partner</h1>
+        <hr class="align-self-center red-hr" />
```

Comment:
- [ ] It will be better 😃 to use one heading tag of h1 on the page that's for semantic reason as you see here I see there is two h1 on the page, so please try to organize the page and use one of the tag of h1 in every page you can use after one of h1 you can use the other hierarchy h23... 😊